.github/CODEOWNERS

```diff
@@ -2,36 +2,39 @@
 # Syntax and details are here: https://help.github.com/articles/about-codeowners/
 #
 # Global code ownership, fallback
-*                   @kad @mythi @bart0sh @rojkov
+*                   @kad @mythi @bart0sh
 
 # CI and building
-.github/workflows/  @rojkov @bart0sh @kad @mythi
-Jenkinsfile         @rojkov @bart0sh @kad @mythi
-Makefile            @rojkov @bart0sh @kad @mythi
-/build/             @rojkov @bart0sh @kad @mythi
-/scripts/           @rojkov @bart0sh @kad @mythi
+.github/workflows/  @bart0sh @kad @mythi
+Jenkinsfile         @bart0sh @kad @mythi
+Makefile            @bart0sh @kad @mythi
+/build/             @bart0sh @kad @mythi
+/scripts/           @bart0sh @kad @mythi
 
 # Binaries
-/cmd/               @rojkov @bart0sh
-/cmd/fpga_tool/     @bart0sh @kad @rojkov
-/cmd/gpu_plugin/    @rojkov @bart0sh @uniemimu
-/cmd/qat_plugin/    @rojkov @mythi
-/cmd/vpu_plugin/    @rojkov @mythi
+/cmd/               @mythi @bart0sh
+/cmd/fpga_tool/     @bart0sh @kad @mythi
+/cmd/gpu_plugin/    @tkatila @bart0sh @uniemimu
```

Comment:
What about gpu_nfdhook ?

Reply:
@tkatila any proposals? as far as I understand cmd/ owners have access to everything under cmd/, so @mythi and @bart0sh are owners of gpu_nfdhook by default.

Reply:
I added gpu_nfdhook same as gpu_plugin.